assignments/number-functions/code.js

```diff
@@ -0,0 +1,10 @@
+const add= (a,b)=> a+b
+const subtract= (a,b)=> a-b
+const multiply= (a,b)=> a*b
+const divide= (a,b)=> a/b
+const averageOfTwo= (a,b)=> (a+b)/2
+const averageOfThree= (a,b,c)=> (a+b+c)/3
+const distance= (a,b)=> Math.abs(a-b)
+const manhattanDistance= (x1,x2,y1,y2)=> Math.abs(x1-y1)+Math.abs(x2-y2)
```

Comment:
This is a little odd. While it works you the way you named the arguments doesn't match up with their meaning. It would be better like this:

```suggestion
const manhattanDistance = (x1, y1, x2, y2)=> Math.abs(x1 - x2)+Math.abs(y1 - y2);
```

These names indicate to the reader that the first two arguments are one x,y point and the third and fourth are another x,y point. 

I see from the history of your changes that you originally had the right variable names but were subtracting the wrong values, e.g. x1 - y1. When you noticed that, you should have left the names in the argument list as they were and changed the code to subtract the x values from each other and the y values from each other.


You could also make this even clearer by using the `distance` function you just defined:

```suggestion
const manhattanDistance = (x1, y1, x2, y2)=> distance(x1, x2) + distance(y1, y2);
```

This capture the idea of the definition of Manhattan distance: it's the sum of the distances on each axis.

Reply:
just finished fixing that

Reply:
You still have the same problem in `euclideanDistance`.